django_mongodb_backend/operations.py

```diff
@@ -195,28 +195,6 @@ def execute_sql_flush(self, tables):
             if not options.get("capped", False):
                 collection.delete_many({})
 
-    def prep_lookup_value(self, value, field, lookup):
-        """
-        Perform type-conversion on `value` before using as a filter parameter.
-        """
-        if getattr(field, "rel", None) is not None:
-            field = field.rel.get_related_field()
-        field_kind = field.get_internal_type()
-
-        if lookup in ("in", "range"):
-            return [
-                self._prep_lookup_value(subvalue, field, field_kind, lookup) for subvalue in value
-            ]
-        return self._prep_lookup_value(value, field, field_kind, lookup)
-
-    def _prep_lookup_value(self, value, field, field_kind, lookup):
-        if value is None:
-            return None
-
-        if field_kind == "DecimalField":
-            value = self.adapt_decimalfield_value(value, field.max_digits, field.decimal_places)
-        return value
-
```

Comment:
Why is this no longer relevant?

Reply:
Possibly a [remnant of mongoengine](https://github.com/mongodb/django-mongodb-backend/pull/2#discussion_r1590188184).

Reply:
I put details about some of the changes in the commit messages. In this case: "Remove special-casing of DecimalField calling adapt_decimalfield_value() obsoleted by https://github.com/django/django/commit/1860a1afc9ac20750f932e8e0a94b32d096f2848"